src/d3a/constants.py

```diff
@@ -46,3 +46,6 @@
 D3A_TEST_RUN = False
 
 IS_CANARY_NETWORK = False
+CN_PROFILE_EXPANSION_DAYS = 7
+
+RUN_IN_REALTIME = IS_CANARY_NETWORK
```

Comment:
if `RUN_IN_REALTIME ` is supposed to follow `IS_CANARY_NETWORK ` then why not simply having one of the flag?

Reply:
In order to test the behaviour of the strategies, if `IS_CANARY_NETWORK=True` you want to disable the `RUN_IN_REALTIME` here manually and locally. We have to discuss how to rerun the strategy tests with `IS_CANARY_NETWORK=True` on travis.

Reply:
I just started the PR, but judging from this I guess that you essentially use IS_CANARY_NETWORK in order to control whether the strategy needs to recycle its profiles. I assume that when you say strategy tests you are talking about the unit tests, right ? In the unit tests though the simulation.py is not invoked, therefore the `RUN_IN_REALTIME` is not really needed. I think I have misunderstood something though.

Reply:
I meant the integrationtests that test the strategies (correct bidding, energy consumption, energy provision, following the provided profiles). As i added this rotation of profiles, it might be needed.

Reply:
ATM this should be dealt with. Variables RUN_IN_REALTIME and IS_CANARY_NETWORK should be completely independent, therefore we should be able to run the tests with IS_CANARY_NETWORK=True without having to wait for ages.